### PR TITLE
Closes #6442: GitLab CI failure

### LINKED ISSUE #6442: GitLab CI pipeline failure when using official Docker images
### Environment
* Nautobot version (Docker tag too if applicable): 2.3.9
* Python version: 3.11
* Database platform, version: postgres 14
* Middleware(s): GitLab Enterprise 17.1

### Steps to Reproduce
1. Create a .gitlab-ci.yml that uses any official `networktocode/nautobot[-dev]` image.
2. Run the CI pipeline.

### Expected Behavior

Pipeline should build & run the container and hand off to any scripts/actions as defined in `.gitlab-ci.yml`.

### Observed Behavior

Pipeline fails after init:

```
💡 Superuser Username: admin, E-Mail: admin@example.com
[: 1: Syntax error: end of file unexpected (expecting "then")
Cleaning up project directory and file based variables
00:01
ERROR: Job failed: exit code 2
```

### PR DESCRIPTION
**Pull request by austind**
# Closes #6442
# What's Changed

Quote exec var in `docker_entrypoint.sh`. Verified in testing with a custom image that this resolves the GitLab CI issue.

Appears related to [this GitLab Runner issue](https://gitlab.com/gitlab-org/gitlab-runner/-/issues/4140).

# Screenshots

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example App Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Review by glennmatthews — COMMENTED** [review]
Please add a `changes/6442.fixed` change fragment file.

**Comment by austind** [comment]
> Please add a `changes/6442.fixed` change fragment file.

Done!

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -96,4 +96,4 @@ fi
 # (i.e. the RUN instruction in the Dockerfile)
 #
 # shellcheck disable=SC2068
```
Comment (glennmatthews) [inline docker/docker-entrypoint.sh:98]: It was pointed out to me that we can remove this line as the below change is exactly what shellcheck was trying to tell us to do 🤦.

**Review by glennmatthews — APPROVED** [review]

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (austind) [reply docker/docker-entrypoint.sh]: Done!

**Pull request merged**